README.md

```diff
@@ -320,6 +327,66 @@ Chewy.settings = {
   User.plugin :chewy_observe           # just for User
   ```
 
+#### Example of data request
+
+You can use the next query:
+
+```ruby
+def search
+  @users = UsersIndex.query(query_string: { fields: [:first_name, :last_name, :email, ...], query: search_params[:query], default_operator: 'and' })
+  render json: @users.to_json, status: :ok
+end
+```
+
+And add new user via Rails console:
+
+```ruby
+User.create(
+  first_name: "test1",
+  last_name: "test1",
+  email: 'test1@example.com',
+  # other fields
+)
+```
+
+It creates User index too:
+
+```ruby
+...
+  UsersIndex::User Import (355.3ms) {:index=>1}
+=> #<User id: 1, first_name: "test1", last_name: "test1", email: "test1@example.com", # other fields>
+```
+
+By `http://localhost:3000/users/search?query=test1@example.com` request you will get the following response:
```

Comment:
When issuing ....

Reply:
Thank you. I combined it with Dalton's comment https://github.com/toptal/chewy/pull/773#discussion_r595390417